.github/workflows/check.yml

```diff
@@ -41,3 +41,5 @@ jobs:
           | jq '.packages | .[] | .name' \
           | rg --invert-match 'atleta-(node|runtime)' \
           | xargs -I {} cargo clippy --package {} --all-targets --all-features -- --deny warnings --deny missing_docs
+    - name: Run shellcheck
+      run: fdfind --type executable | xargs -I {} shellcheck {}
```

Comment:
JFYI `fd-find` has a `--exec` option which behave the same as `xargs`, but with some additional tweaks of substitution of matched file path.

Reply:
Ok, this command looks simpler. 
`fdfind --type executable --exec shellcheck {}`
Tested, it works. Thanks for the comment